sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_file_client.py

```diff
@@ -502,22 +503,18 @@ def flush_data(self, offset,  # type: int
         except StorageErrorException as error:
             process_storage_error(error)
 
```

Comment:
Should we also update any samples @xiafu-msft added recently?

Reply:
@xiafu-msft I think the upload/download sample I updated is the only one?